x-pack/plugins/lens/public/editor_frame_service/embeddable/embeddable.tsx

```diff
@@ -110,7 +110,9 @@ export class Embeddable
 
     this.expressionRenderer = deps.expressionRenderer;
     this.initializeSavedVis(initialInput).then(() => this.onContainerStateChanged(initialInput));
-    this.subscription = this.getInput$().subscribe((input) => this.onContainerStateChanged(input));
+    this.subscription = this.getUpdated$().subscribe(() =>
+      this.onContainerStateChanged(this.input)
```

Comment:
nit: Should `this.input` be used? There's also `this.getInput()` which is used in other places. Not sure whether important, just wanted to point it out.

Reply:
`this.getInput()` is public and `this.input` is protected, so inside embeddable it is fine to use both